subworkflows/local/allele_specific_calling.nf

```diff
@@ -14,13 +14,13 @@ include { TABIX_TABIX          } from '../../modules/nf-core/tabix/tabix/main'
 
 workflow ALLELE_SPECIFIC_CALLING {
     take:
-        ch_ind_vcf_tbi // channel (mandatory): [ val(meta), [ path(vcf), path(tbi) ] ]
-        ch_bam_bai     // channel (mandatory): [ val(meta), [ path(bam), path(bai) ] ]
-        ch_fasta       // channel (mandatory): [ path(fasta) ]
-        ch_fai         // channel (mandatory): [ path(fai) ]
-        ch_dict        // channel (mandatory): [ val(meta), [ path(dict) ] ]
-        ch_intervals   // channel (mandatory): [ path(intervals) ]
-        ch_case_info   // channel (mandatory): [ val(case_info) ]
+        ch_ind_vcf_tbi // channel: [mandatory] [ val(meta), [ path(vcf), path(tbi) ] ]
+        ch_bam_bai     // channel: [mandatory] [ val(meta), [ path(bam), path(bai) ] ]
+        ch_fasta       // channel: [mandatory] [ val(meta), path(fasta) ]
+        ch_fai         // channel: [mandatory] [ val(meta), path(fai) ]
+        ch_dict        // channel: [mandatory] [ val(meta), path(dict) ]
+        ch_intervals   // channel: [mandatory] [ path(intervals) ]
```

Comment:
If you're updating GATK4_ASEREADCOUNTER, consider giving ch_intervals a meta too for the full set? 😃

Reply:
I think you are only supposed to add meta if the file is an actual reference, but maybe we can have a second opinion, I might be wrong @jemten do you know what the standards are?

Reply:
Don't know if the community has reached a consensus on adding meta to interval files and such.